e2e-tests/adapters/cypress/e2e/remote-file.cy.ts

```diff
@@ -29,7 +29,7 @@ const configs = [
   {
     title: `remote-file (SSR, Page Query)`,
     pagePath: `/routes/ssr/remote-file/`,
-    placeholders: false,
+    placeholders: true,
```

Comment:
This test was disabled because of problems that this PR is fixing. In context of this tests those problems weren't fatal, however it meant that image placeholders were just not working.

The fix in this PR also solves other problems than just placeholders, but this was existing test that already we had